apps/news-site/news-site-next/src/data/index.js

```diff
@@ -28,28 +56,55 @@ import * as formsAr from "@/data/ar/form";
 
 export const dataSource = {
     en: {
-        content: contentEn,
-        settings: settingsEn,
-        footer: footerEn,
-        sitemap: sitemapEn,
+        alerts: alertsEn,
+        pages: pagesEn,
+        sections: sectionsEn,
+        content: {
+            home: homeEn,
+            us: usEn,
+            world: worldEn,
+            politics: politicsEn,
+            business: businessEn,
+            opinion: opinionEn,
+            health: healthEn,
+        },
+        language: languageEn,
         buttons: buttonsEn,
         links: linksEn,
         forms: formsEn,
     },
```

Comment:
this could probably be deduplicated at some point with dynamic imports ?

Reply:
yup, each page will fetch it's own data in just the language it needs